sdk/search/Azure.Search/api/Azure.Search.netstandard2.0.cs

```diff
@@ -0,0 +1,276 @@
+namespace Azure.Search
```

Comment:
Interesting....  this is probably my fault.  There's no problem adding it here.

Reply:
This is what I get if I run Export-API.ps1, I've noticed this has been happening for the last week or so on the master branch.

Reply:
Removed, since now we are failing Search CI.